model/docs/getting_started.qmd

```diff
@@ -204,6 +204,27 @@ Now, let's investigate the output, particularly the posterior distribution of th
 out = model1.plot_posterior(var="Rt")
 ```
 
+We can use ArviZ package to create model diagnostics and visualizations. Below we use `plot_trace` to inspect the trace of daily $R_t$ estimates.
+
+```{python}
+#| label: viz_trace
```

Comment:
These ArviZ figures don't seem to have been generated by the build in CI. The ArviZ figures in `example_with_datasets.qmd` do seem to have been generated.

Reply:
I am not sure what causes this or how to fix!?

Reply:
Resolved below; thanks @gvegayon https://github.com/CDCgov/multisignal-epi-inference/pull/211#issuecomment-2192036047